gulpfile.js

```
@@ -60,24 +60,45 @@ const pluginGlobs = [
-            const fileName = findNewPDEPlugin(extensionString.substring(extensionString.lastIndexOf('/') + 1, ind));
             
             if (ind >= 0) {
-                return extensionString.substring(0, extensionString.lastIndexOf('/') + 1) + fileName;
+                const baseName = extensionString.substring(extensionString.lastIndexOf('/') + 1, ind);
+                const fileName = findNewPDEPlugin(baseName);
+                
+                if (fileName) {
+                    // update bundle with version from "/server"
+                    return extensionString.substring(0, extensionString.lastIndexOf('/') + 1) + fileName;
+                }else{
+                    console.log('not found:' + baseName);
```

ESLint

[error] 'console' is not defined.

(no-undef)